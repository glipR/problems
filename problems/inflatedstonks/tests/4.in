20
RLDDDLUDRLDURRRRLUUD
ULDURDLDLUDRRLRDUDRR
DLURRRDDUUDRURRULDDD
LURUULLLUDDURUDRLDUD
LURLRRLDRLULUULDRLRU
LURUDUDRURDLRLURLLUR
URUDUDDURUULDLURDRLR
UURLUUURURULURUDLUUD
LUDDUULRLDULLRRLDLLD
LRUDDRLDUURLULUDDULR
ULLDRRLULLRLRRLDLUDU
RRURDRDRUDRRUDDULLRD
LUDLDDURURRULUURDRRL
RDLURDUDRUULUURDRURL
LDRRLUURRLULLUDDDULD
ULURDULRDLULDDURDURD
LLDLUDRDLDLUULRDUUDL
UUUDDRDDRLLRDDDUUDLL
ULLRRRRLLRULUDLRRRLD
RLRDRDRRRRRUDRLLRLLR
391224 49899 511122 189400 262571 66085 299878 649832 637847 443284 24845 51055 842793 582522 517028 320702 428093 528654 958222 853676
311999 928032 933922 771095 247422 118247 329191 488060 249244 925140 386585 664879 628637 890156 801788 379862 970541 388694 468843 214817
3319 87819 585895 201176 197525 198563 357719 688337 397751 760784 755377 775116 224636 903631 271907 254527 634779 2187 656810 417015
117113 287040 31832 633642 736604 438797 320349 268947 90962 531313 724624 777690 277853 395247 803049 445075 854036 221040 800868 964955
698260 199691 982789 618418 856161 318753 931521 636498 384126 643555 598915 463381 944698 862204 607330 646619 260232 582811 852240 142676
920200 155235 277465 614591 757009 269891 329953 746366 373025 169679 190661 208338 674666 231967 753898 35864 494525 164644 632958 100046
696494 304695 691729 636228 610571 130961 514251 178732 352274 846531 594888 313234 691259 113134 754553 825367 84871 225454 676842 588883
166499 113854 534724 224680 673650 131372 982895 611085 33232 832181 4493 246843 971040 971406 549142 493602 608513 283577 863384 42955
145228 168126 377446 479530 90708 565636 866971 274835 34085 864379 643826 551243 306894 340413 136409 701249 247457 992208 600177 814369
586586 647706 902379 483305 718052 571171 774531 325550 440288 631162 539659 11044 218519 766286 431507 823707 863496 744771 137754 199552
574620 277311 658963 414945 333578 379557 871186 453139 48638 411420 309333 983085 319897 875867 315629 420738 215038 65849 439068 525323
519118 676840 509359 288618 474733 391318 369865 778601 155184 852358 655449 397219 827502 43300 931830 572124 748157 202616 500750 138879
284184 434854 881290 984050 834391 528497 537483 426916 676219 986257 195748 381333 217704 89372 203141 631740 838809 173588 452181 701301
789862 498571 767793 662046 968303 728944 770291 945309 10839 60024 270565 7998 281317 269489 308829 699734 553764 774765 933638 349443
456182 345884 485944 943443 76337 970256 19804 305562 600098 394373 397991 783329 939433 864484 723520 50090 514940 817345 623455 581247
385226 778320 37523 968569 522084 721838 128990 122962 346558 386396 691128 524274 842198 173430 628353 427899 977457 742952 202192 774310
927403 631744 521076 343089 970993 26088 53369 27153 682340 588402 786182 340270 610988 889235 829009 22357 648528 513416 725757 669548
153945 407978 272707 159023 745557 110315 278651 617266 697343 285550 458571 637188 879618 999246 288307 26413 413242 332152 470988 417724
643751 384778 560252 970140 976146 531432 807929 290588 609087 394333 80282 679290 501716 519017 203825 554369 866490 645284 863750 635053
71954 430512 674313 413822 706751 732289 225008 765495 269520 710158 215825 285784 707797 159510 916694 652915 749949 149201 385169 219239