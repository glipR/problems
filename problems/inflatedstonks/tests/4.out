139
1 1
0 3
0 6
2 9
0 11
0 12
1 16
0 17
0 18
0 2
3 11
1 11
7 13
3 14
4 12
1 14
2 16
0 19
1 18
3 8
0 7
0 5
1 6
3 2
2 18
5 1
0 10
4 11
3 15
5 18
2 19
4 0
8 1
4 4
5 5
4 7
6 7
3 9
1 17
7 0
8 4
9 13
7 9
7 11
7 7
5 11
6 11
5 12
4 15
5 17
5 19
8 6
5 6
6 13
7 18
7 17
6 19
6 3
8 0
9 1
9 8
8 9
6 15
8 13
9 14
11 0
12 1
9 5
12 7
11 7
10 9
9 10
7 19
10 19
9 4
10 10
16 11
14 12
13 11
13 8
11 9
13 12
11 10
9 18
10 17
8 18
9 15
11 17
14 6
11 5
14 13
11 14
12 0
16 17
14 18
12 15
10 18
8 3
12 5
15 6
13 18
13 3
13 0
15 2
13 7
14 14
15 3
15 13
13 14
14 16
17 16
18 2
17 1
16 3
16 14
19 6
18 9
16 10
17 11
17 3
17 4
16 5
15 9
15 7
16 6
18 8
14 19
15 18
17 19
18 15
19 0
19 2
19 4
19 12
17 14
17 13
19 15
19 18
18 19
